functions/supplier/debezium-supplier/src/test/java/org/springframework/cloud/fn/supplier/debezium/custom/DebeziumEngineBuilderAutoConfigurationTest.java

```diff
@@ -57,8 +57,8 @@
  */
 @Tag("integration")
 @Testcontainers
-public class DebeziumEngineAutoConfigurationTest {
-	private static final Log logger = LogFactory.getLog(DebeziumEngineAutoConfigurationTest.class);
+public class DebeziumEngineBuilderAutoConfigurationTest {
+	private static final Log logger = LogFactory.getLog(DebeziumEngineBuilderAutoConfigurationTest.class);
```

Comment:
Switching to `LogAccessor` has some benefits, such as it auto-checking if the logging is at the requested level.

Reply:
Ha, @artembilan just pointed out to me that this my comment was in the test code (yeh, I don't care which logger we use here and really not typical to have a logger in the test code).